lib/const.rb

```diff
@@ -104,15 +102,13 @@
 CREW_DEST_MAN_PREFIX  = File.join(CREW_DEST_DIR, CREW_MAN_PREFIX)
 
 # Local constants for contributors.
-CREW_LOCAL_REPO_ROOT = `cd #{CREW_LIB_PATH} && git rev-parse --show-toplevel 2> /dev/null`.chomp
+CREW_LOCAL_REPO_ROOT = `git rev-parse --show-toplevel 2> /dev/null`.chomp
+CREW_LOCAL_BUILD_DIR = "#{CREW_LOCAL_REPO_ROOT}/release/#{USER_SPACE_ARCH}"
```

Comment:
My problem here is that this requires the build to be started in a git dir, and that's not my workflow. I start my build in a tmpdir outside of a git tree.

This PR ensures that manifest files are only copied out of the build into `#{CREW_LOCAL_REPO_ROOT}/manifest/`, but then `CREW_LOCAL_REPO_ROOT` is only defined if the crew is run inside a git dir...

If crew build isn't started inside a git clone of a chromebrew repo, builds just don't work, right?